### PR TITLE
Fix empty pages dark theme

### PR DESCRIPTION
**Pull request by aishwarya-bm**
This PR contains minor changes related to :

1. view for empty pages & error page
2. MUI dark theme
3. minor code changes/refactoring to fix bugs

### CODE REVIEW

**Review by shubhamkumar648 — APPROVED** [review]
- all functionality is working which is given in pr 
- remaining all code looks neat and readable 
- good to merge

File: src/App.js [hunk 1]
```diff
@@ -2,41 +2,53 @@ import { RoutePaths } from "RoutePaths";
...
-    fontWeightMedium: 500,
-  },
-});
+// export const theme = createTheme({
+//   palette: {
+//     mode:  "light" ,
+//     primary: {
+//       main: "#407BFF",
+//     },
+//     secondary: {
```
Comment (nishasen) [inline src/App.js:16]: remove the commented code

**Review by coldpigli — APPROVED** [review]
Everything looks good. Go ahead an merge

**Review by nishasen — APPROVED** [review]
Everything looks good. Remove the commented code in index.js

**Pull request merged**